ods_ci/utils/scripts/Sender/EmailSender.py

```diff
@@ -24,16 +24,16 @@ def __init__(self):
         self._message = MIMEMultipart()
 
     def prepare_payload(
-        self, text: str = "", attachments: Optional[List[Any]] = None
+        self, text: str = "", attachments: list[Any] | None = None
```

Comment:
why?

Reply:
linter wants this when you have a py3.11 project

at some point they made list[...] be an alias for typing.List[...], so that's a cleanup it wants

not sure about why it dislikes Optional, that should be fine imo, and | None is not a clear win in my eyes